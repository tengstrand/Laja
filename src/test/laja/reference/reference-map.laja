#write "target/test-output/reference-map.out"
#set (map1 = { avalue: "value of a"})
#set (map2 = { a: "a" b: { c: "ccc" } })
The value of map1.avalue is: {map1.avalue}
map2.b.c = {map2.b.c}
#end
