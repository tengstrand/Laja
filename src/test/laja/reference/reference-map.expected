The value of map1.avalue is: value of a
map2.b.c = ccc
